x-pack/plugins/security_solution/public/detections/components/rules/step_define_rule/index.tsx

```diff
@@ -372,6 +390,24 @@ const StepDefineRuleComponent: FC<StepDefineRuleProps> = ({
             }}
           />
         </Form>
+        {ruleType !== 'machine_learning' && ruleType !== 'threat_match' && (
+          <>
+            <EuiSpacer size="s" />
+            <PreviewQuery
+              dataTestSubj="something"
+              idAria="someAriaId"
```

Comment:
Did you want data test subject of "something"? and idAria to be "someAriald" or something more descriptive

```ts
dataTestSubj="something"
idAria="someAriaId"
```

Reply:
These are already pretty descriptive in my opinion 😉 Thanks for the find - cleaned up here https://github.com/elastic/kibana/pull/79947